content/en/docs/concepts/services-networking/service.md

```diff
@@ -642,9 +642,9 @@ You must enable the `ServiceLBNodePortControl` feature gate to use this field.
 
 #### Specifying class of load balancer implementation {#load-balancer-class}
 
-{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.21" state="beta" >}}
 
-Starting in v1.21, you can optionally specify the class of a load balancer implementation for
+From Kubernetes v1.21 onwards, you can optionally specify the class of a load balancer implementation for
 `LoadBalancer` type of Service by setting the field `spec.loadBalancerClass`.
 By default, `spec.loadBalancerClass` is `nil` and a `LoadBalancer` type of Service uses
 the cloud provider's default load balancer implementation. 
```

Comment:
You _can_ do this. Do we recommend that you should? Or is it truly optional?

Reply:
This is to provide a capability for users to disable default load balancer. 
So, this is truly optional. 
And it is for the users who want to use other load balancer implementations, they can set this value to disable default cloud providers' load balancer implementation.

Reply:
@sftim do you have any comments on this, or can we move forward and merge this?

Reply:
Let's explain that this change enables you to use a load balancer technology other than the cloud provider default. I don't think that's obvious.